packages/cli/generators/app/templates/index.ts

```diff
@@ -0,0 +1,10 @@
+// DO NOT EDIT THIS FILE
+// Add any aditional (re)exports to src/index.ts instead.
+export * from './src';
+
+const application = (module.exports = require('./src'));
```

Comment:
My two comment above advocating for preservation of package-level `index.js` file apply to this place too.